pennylane/drawer/tape_mpl.py

```diff
@@ -271,19 +262,66 @@ def _tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, *, fig
         for op in layer_ops:
             _add_operation_to_drawer(op, drawer, layer, config)
 
-    for wire in _get_measured_wires(tape.measurements, list(range(n_wires))):
-        drawer.measure(n_layers, wire)
+    if measurements == "dots":
+        for wire in wire_map.values():
+            drawer.ax.text(
+                n_layers + 0.25,
+                wire,
+                s="...",
+                ha="center",
+                va="center_baseline",
+                fontsize=1.5 * drawer.fontsize,
+            )
+    else:
+        for wire in _get_measured_wires(measurements, list(range(n_wires))):
+            drawer.measure(n_layers, wire)
 
-    measured_bits = _get_measured_bits(tape.measurements, bit_map, drawer.n_wires)
-    if measured_bits:
-        drawer.measure(n_layers, measured_bits)
+        measured_bits = _get_measured_bits(measurements, bit_map, drawer.n_wires)
+        if measured_bits:
+            drawer.measure(n_layers, measured_bits)
 
     return drawer.fig, drawer.ax
 
 
+def _tape_mpl(tape, wire_order=None, show_all_wires=False, max_length=None, **kwargs):
+    """Private function wrapped with styling."""
```

Comment:
We might need a clearer docstr here to describe the main dispatching logic below and hint on some trick (e.g. reason why different wire_map 's in use) so that we can easily understand in the future maintanance

Reply:
Any more information about what's confusing? I think I'm too familiar with this code that it's a blind spot.

Reply:
I'll try draft some and make a suggest